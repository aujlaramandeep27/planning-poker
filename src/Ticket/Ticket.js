import React from 'react';
import {Link} from "react-router-dom";

export default function Ticket() {

    return (
        <div className="uk-flex uk-flex-center">
            <div className="uk-card uk-card-default uk-card-body uk-width-1-2@m">
                <h1 className="uk-heading-small">Add a Ticket</h1>
                <input placeholder="Ticket #" className="uk-input uk-margin"></input>
                <Link to="/host/" className="uk-button uk-button-primary">CREATE</Link>
            </div>
        </div>
    );
};