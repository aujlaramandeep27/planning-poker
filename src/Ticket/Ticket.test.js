import Ticket from "./Ticket";

it('Ticket renders correctly', () => {
    expect(Ticket()).toMatchSnapshot();
});