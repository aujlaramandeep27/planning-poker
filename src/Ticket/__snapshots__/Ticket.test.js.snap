// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticket renders correctly 1`] = `
<div
  className="uk-flex uk-flex-center"
>
  <div
    className="uk-card uk-card-default uk-card-body uk-width-1-2@m"
  >
    <h3
      className="uk-heading-medium uk-text-center"
    >
      Add a Ticket
    </h3>
    <a
      href="#/host/"
    >
      <span
        className="uk-position-small uk-position-top-left"
        uk-icon="icon: arrow-left; ratio: 3"
      />
    </a>
    <input
      className="uk-input uk-margin uk-width-1-2@m uk-align-center"
      onChange={[Function]}
      placeholder="Ticket #"
    />
    <a
      className="uk-width-1-2@m uk-align-center"
      href="#/host/"
    >
      <button
        className="uk-button uk-button-primary uk-width-1-1@m"
        disabled={true}
      >
        CREATE
      </button>
    </a>
  </div>
</div>
`;
